pyramid_swagger/api.py

```diff
@@ -101,6 +101,29 @@ def view_for_api_declaration(request):
     return view_for_api_declaration
 
 
+def resolve_ref(spec, url):
```

Comment:
Included an entry for pyramid_swagger.dereference_served_schema in configuration.rst